src/pages/developing/svelte-tutorial/step-1.mdx

```diff
@@ -0,0 +1,486 @@
+directory.
+
+```bash
+git clone [your fork SSH/HTTPS]
+cd carbon-tutorial-svelte
+```
+
+### Add upstream remote
+
+Add a remote called `upstream` so we can eventually submit a pull request once
+you have completed this tutorial step.
+
+```bash
+git remote add upstream git@github.com:carbon-design-system/carbon-tutorial-svelte.git
+```
+
+Or, if you prefer to use HTTPS instead of SSH with your remotes:
+
+```bash
+git remote add upstream https://github.com/carbon-design-system/carbon-tutorial-svelte.git
+```
+
+Verify that your forked repository remotes are correct:
+
+```bash
+git remote -v
+```
+
+Your terminal should output something like this:
+
+```bash
+origin	[your forked repo] (fetch)
+origin	[your forked repo] (push)
+upstream	git@github.com:carbon-design-system/carbon-tutorial-svelte.git (fetch)
+upstream	git@github.com:carbon-design-system/carbon-tutorial-svelte.git (push)
+```
+
+### Branch
+
+Now that we have our repository set up, let's check out the branch for this
+tutorial step's starting point. Run the two commands:
+
+```bash
+git fetch upstream
+git checkout -b svelte-step-1 upstream/svelte-step-1
+```
+
+## Build and start
+
+We have the repository forked to your GitHub account, cloned down to your
+machine, and the starting branch checked out. Next, install the Svelte app's
+dependencies with:
+
+```bash
+yarn
+```
+
+After the dependencies are installed, you can start the app with:
+
+```bash
+yarn dev
+```
+
+Your default browser should open up with an empty page that says:
+`Hello Carbon! Well, not quite yet. This is the starting point for the Carbon tutorial.`
+
+## Install Carbon
+
+Even though we installed existing dependencies, we've yet to install the Carbon
+packages.
+
+- `carbon-components-svelte` - Svelte components
+- `carbon-preprocess-svelte` - Carbon Svelte preprocessors
+
+Stop your development server with `CTRL-C` and install Carbon dependencies with:
+
+```bash
+yarn add -D carbon-components-svelte carbon-preprocess-svelte
+```
+
+## Configure svelte.config.js
+
+SvelteKit uses [vite](https://github.com/vitejs/vite) as a development server.
+We need to configure vite to optimize dependencies from
+`carbon-components-svelte` that do not support ES Modules (ESM).
+
+In addition, we will utilize the recommended Carbon preprocessor preset. The
+preset includes two preprocessors:
+
+- **optimizeImports**: rewrites Carbon Svelte imports to their source path in
+  the script block
+- **elements**: computes Carbon theme tokens in the style block
+
+```js path=./svelte.config.js
+import adapter from '@sveltejs/adapter-static';
+import * as carbon from 'carbon-preprocess-svelte';
+
+/** @type {import('@sveltejs/kit').Config} */
+export default {
+  preprocess: carbon.presetCarbon(),
+  kit: {
+    target: '#svelte',
+    adapter: adapter(),
+    vite: {
+      optimizeDeps: { include: ['clipboard-copy'] },
+    },
+  },
+};
+```
+
+Then, start the app again.
+
+```bash
+yarn dev
+```
```

Comment:
I haven't been able to succesfully run the dev server after applying these configurations:
<img width="1231" alt="image" src="https://user-images.githubusercontent.com/40550942/200341323-5806dee0-e231-426b-88c8-e1165849cfab.png">

Branch I'm working on: https://github.com/francinelucca/carbon-tutorial-svelte/tree/svelte-step-1

Reply:
CC @metonym